server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/mail/Email.scala

```diff
@@ -84,14 +86,28 @@ object Email {
         bodyStructure = bodyStructure,
         textBody = bodyStructure.textBody,
         htmlBody = bodyStructure.htmlBody,
-        attachments = bodyStructure.attachments)
+        attachments = bodyStructure.attachments,
+        asEmailHeaders(mime4JMessage.getHeader)
+      )
     }
   }
+
+  private def asEmailHeaders(header: Header): List[EmailHeader] = {
+    header.iterator()
+      .asScala
+      .map(header => EmailHeader(
+        EmailHeaderName(header.getName),
+        EmailHeaderValue(new String(header.getRaw.toByteArray)
+          .replace(header.getName + ":", ""))))
```

Comment:
I prefer using `new String(header.getRaw.toByteArray, CHARSET).subString(header.getName.length + 1)`

Your code is buggy for instance with `header: header: value` whose value should be ` header: value` and not `  value`